emacs-live-py-mode/live-py-mode.el

```diff
@@ -147,6 +147,7 @@ When the source buffer is narrowed the trace buffer remains
 aligned but will not hide the part after the narrowing."
   (let* ((output-window (get-buffer-window live-py-trace-name))
          (point-min-pos (point-min))
+         (scroll-margin 0)
```

Comment:
I suggest

  `(let ((scroll-margin 0)) (recenter-top-bottom 0))`

instead to make it clear where the setting is needed.